boards/arm/frdm_k22f/doc/frdm_k22f.rst

```diff
@@ -0,0 +1,285 @@
+.. _frdm_k64f:
```

Comment:
This looks to be a copy of the frdm_k64f.rst file, renamed as frdm_k22f.rst

The document label at the top of the file needs to be unique across all the documents, and the title (and content) below should be updated to document the FRDM-K22F board (including the image reference).

Reply:
Did not plan to prioritze the documentaiton for now, but now it's fixed :-D